website/content/docs/v0.15/Guides/troubleshooting-control-plane.md

```diff
@@ -418,10 +474,12 @@ spec:
 
 Control plane nodes have enough secret material to generate `apid` server certificates, but worker nodes
 depend on control plane `trustd` services to generate certificates.
-Worker nodes wait for `kubelet` to join the cluster, then `apid` queries Kubernetes endpoints via control plane
-endpoint to find `trustd` endpoints, and use `trustd` to issue the certficiate.
+Worker nodes wait for their `kubelet` to join the cluster.
```

Comment:
we should probably update the doc here, as with cluster discovery enabled (which is the default), worker nodes find the control plane nodes via the cluster discovery, so they can join on `apid` level before the `kubelet` is ready

Reply:
:+1: but probably needs more (and broader) work than this PR